src/platform/plugins/shared/es_ui_shared/public/components/json_editor/json_editor.tsx

```diff
@@ -87,6 +90,7 @@ function JsonEditorComp<T extends object = { [key: string]: any }>({
       {...rest}
     >
       <CodeEditor
+        aria-describedby={jsonEditorRowId}
```

Comment:
@eokoneyo I think it's extra, EuiFormRow automatically populate aria-describedby for the first child. Please revert that file